common/source/docs/common-planning-a-mission-with-waypoints-and-events.rst

```diff
@@ -159,8 +159,25 @@ There is also a Copter-specific :ref:`Copter Mission Command List <copter:missio
 Mission Reset
 =============
 
+On Copter and Rover, the mission is reset to the beginning of the mission list every disarm. On Plane, it is reset similarly, except in AUTO mode which does not reset on disarm and leaves the "mission pointer" on the last mission item to be performed. For all vehicles, any NAV_LAND mission item leaves the "mission pointer" pointing to itself when landing is completed. The :ref:`MIS_OPTIONS<MIS_OPTIONS>` parameter, bit 2, can be set to continue onward in the mission list after landing.
```

Comment:
"mission pointer pointing to itself" is a bit difficult to understand.  Does this mean that it is left pointing at the LAND command?

Reply:
reworded